### ISSUE TITLE
debug_toolbar expects all settings inside DEBUG_TOOLBAR_CONFIG

### ISSUE THREAD

**Issue by aukehuizinga** [jazzband/django-debug-toolbar#1751]
I just came across the toolbar not showing up in our project. I did configure the `INTERNAL_IPS` to also include the IP I was currently using. While debugging by adding print's I figured out the "settings" the debug_toolbar uses are only whatever is in `DEBUG_TOOLBAR_CONFIG`, so to activate the `INTERNAL_IPS` from the project I need to also add:
```Python
DEBUG_TOOLBAR_CONFIG = {
    "INTERNAL_IPS": INTERNAL_IPS
}
```
to the project settings.

The code looking up the settings is:
https://github.com/jazzband/django-debug-toolbar/blob/af96d391ca1890f4b62ec1ca7cd597db27fe2464/debug_toolbar/settings.py#L48-L53

Shouldn't it load all settings instead? Or at least include the `INTERNAL_IPS` also?

**Comment by tim-schilling** [comment]
@aukehuizinga I believe it does load `INTERNAL_IPS` from the project's settings. See https://github.com/jazzband/django-debug-toolbar/blob/5b4450ac521cef1a45bf29e09793ebfed8734159/debug_toolbar/middleware.py#L22

Can you create a reproducible example? If you clone the repo, we have an example project in it.

**Comment by aukehuizinga** [comment]
Ah! Now I see that the settings module from Django is in this file the one called `settings`.
https://github.com/jazzband/django-debug-toolbar/blob/5b4450ac521cef1a45bf29e09793ebfed8734159/debug_toolbar/middleware.py#L8-L11

I think I messed up during debugging expecting the server's IP but it's actually the remote IP (`REMOTE_ADDR` duhhh). Sorry for the confusion.

How I messed up:
Besides the
```
DEBUG_TOOLBAR_CONFIG = {
    "INTERNAL_IPS": INTERNAL_IPS
}
```
I also played around with the replacement function in `DEBUG_TOOLBAR_CONFIG` and bypassed the libraries function.
```
def show_toolbar(_request):
    return DEBUG

DEBUG_TOOLBAR_CONFIG = {
    "INTERNAL_IPS": INTERNAL_IPS
    "SHOW_TOOLBAR_CALLBACK": "project_settings_dev.show_toolbar",
}
```
making me think the `INTERNAL_IPS` thingy inside `DEBUG_TOOLBAR_CONFIG` fixed it.
